<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nhaphang</class>
 <widget class="QMainWindow" name="nhaphang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbxDanhmuc">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phân loại</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>73</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Danh mục</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Số lượng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Giá hàng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã hàng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên hàng</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VNĐ</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thông tin nhập hàng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLuu">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHuy">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hủy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxLoaihang">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Thiết Bị Điện Tử</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phụ Kiện Điện Tử</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TV &amp; Thiết Bị Điện Gia Dụng</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sức Khoẻ &amp; Làm Đẹp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hàng Mẹ, Bé &amp; Đồ Chơi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siêu Thị Tạp Hoá</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hàng Gia Dụng &amp; Đời Sống</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thời Trang Nữ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thời Trang Nam</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Phụ Kiện Thời Trang</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thể Thao &amp; Du Lịch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ôtô, Xe Máy &amp; Thiết Bị Định Vị</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>263</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ngày nhập</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
